README.md

```diff
@@ -98,13 +98,7 @@ The following attributes are available for all models of `viamrtsp` discovery se
 
 ### Next steps
 
-Use the `DiscoverResources` API to retrieve a list of cameras discovered by the service and their configuration. Some cameras will output multiple channels, so review the `rtsp_address` of the cameras to determine which camera streams you wish to add.
-
-## RTSP stream discovery
-### Use `DiscoverComponents`
```

Comment:
should we wait until DiscoverComponents is removed to remove this? fine if no, don't really know how other people use this repo currently.

Reply:
I'd rather remove it - as I did for the docs, to lower usage by external users.